frontend/src/scenes/data-warehouse/editor/sidebar/QueryInfo.tsx

```diff
@@ -151,6 +155,34 @@ export function QueryInfo({ codeEditorKey }: QueryInfoProps): JSX.Element {
                                         loading={updatingDataWarehouseSavedQuery}
                                         options={OPTIONS}
                                     />
+                                    <LemonButton
+                                        type="secondary"
+                                        size="small"
+                                        tooltip="Revert materialized view to view"
+                                        disabledReason={
+                                            savedQuery?.status === 'Running' &&
+                                            'Cannot revert while materialization is running'
+                                        }
+                                        icon={<IconX />}
+                                        onClick={() => {
+                                            if (editingView) {
```

Comment:
Will this button exist when `editingView` be false? if so, whats expected when its clicked?